go-client/client/inputtable_stub.go

```diff
@@ -65,6 +65,8 @@ func (its *inputTableStub) NewAppendOnlyInputTableFromSchema(ctx context.Context
 	if err != nil {
 		return nil, err
 	}
+	// This warns about copying an RWMutex.
```

Comment:
Thought this trick wouldn't work because of:
```go
tbl := &AppendOnlyInputTable{}

tbl2, err := tbl.Head(50) // I thought this line would segfault
```
But since `TableHandle` doesn't embed any structs within it, the table validity check works as expected and there's no issue.
Will do.